src/SFA.DAS.Roatp.CourseManagement.Web/Views/Standards/ViewStandardDetails.cshtml

```diff
@@ -8,6 +8,11 @@
     <a class="govuk-back-link" href="@Model.BackUrl">Back</a>
 }
 
+@if (Model.StandardInformation.IsStandardRegulated && Model.IsApprovedByRegulator!=true)
```

Comment:
`Model.IsApprovedByRegulator == false` or `!Model.IsApprovedByRegulator`

Reply:
Non this is correct, as this is a  nullable boolean, so it the condition is !=true or (==false OR ==null), and !=true is cleaner

Reply:
There is an extension method `GetValueOrDefault` which will return false in case of null.

Reply:
will use that